### DISCUSSION TITLE
ExtremeCloud IQ Cloud Test

### DISCUSSION THREAD

**Discussion by MibenSmmod** [WLAN-Pi/feedback#28] [Feature Request]
I work for Extreme Networks and would like to provide the necessary information to add "Run Extreme XIQ" to the menu.  The first question is how you perform tests.  Is it a simple ping or can it open a socket and test if the port is open?

We have an extensive global cloud management network:  https://cloud-status.extremecloudiq.com/status/index.html

Run Extreme XIQ > 
 - Test Global DC
 - Test RDC: VA
 - Test RDC: VA2
 - Test RDC:  IA-GCP
 - etc. etc.  I can provide the entire menu structure and all IPs and ports for each regional data center

Thank you.
-Mike Rieben, Senior Solution Architect

**Comment by jiribrejcha** [comment]
Hi Mike,

The cloud tests are typically few lines of Python or Shell. Here is how the cloud tests are implemented:
https://github.com/WLAN-Pi/wlanpi-fpms/blob/main/fpms/modules/cloud_tests.py

The menu items are here:
https://github.com/WLAN-Pi/wlanpi-fpms/blob/main/fpms/fpms.py#L510

Many thanks
Jiri

**Comment by joshschmelzle** [comment]
@niklasmato submitted an initial PR for a portion of this. https://github.com/WLAN-Pi/wlanpi-fpms/pull/88

@MibenSmmod feedback appreciated.
